src/matlab.ts

```diff
@@ -46,27 +39,44 @@ export async function setupBatch(platform: string) {
 }
 
 export async function getReleaseInfo(release: string): Promise<Release> {
-    const client: http.HttpClient = new http.HttpClient();
-    const releaseInfo = await client.getJson<MATLABReleaseInfo>(properties.matlabReleaseInfoUrl);
-
-    if (!releaseInfo.result) {
-        return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));
+    // Get release name from input parameter
+    let name: string;
+    if (release.toLowerCase().trim() === "latest") {
+        try {
+            const client: http.HttpClient = new http.HttpClient();
+            const latestResp = await client.get(properties.matlabLatestReleaseUrl);
+            name = await latestResp.readBody();    
+        }
+        catch {
+            return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));
+        }
+    } else {
+        let nameMatch = release.toLowerCase().match(/r[0-9]{4}[a-b]/);
+        if (!nameMatch) {
+            return Promise.reject(Error(`${release} is invalid or unsupported. Specify the value as R2020a or a later release.`));
+        }
+        name = nameMatch[0];
     }
 
-    let name: string = release.toLowerCase().trim().substring(0,6);
-    if (name === "latest") {
-        name = releaseInfo.result.latest;
+    // create semantic version of format year.semiannual.update from release
+    let year = name.slice(1,5);
+    let semiannual = name[5] === "a"? "1": "2";
+    let updateMatch = release.toLowerCase().match(/u[0-9]+/);
+    let version = `${year}.${semiannual}`;
+    let update: string;
+    if (updateMatch) {
+        update = updateMatch[0]
+        version += `.${update[1]}`;
+    } else {
+        // Notify user if Update version format is invalid
+        if (release.trim() !== name) {
+            const invalidUpdate = release.replace(name, "");
+            core.info(`Update version ${invalidUpdate} is invalid. Defaulting to latest update.`);
```

Comment:
@mw-hrastega Would appreciate some feedback on this info message. For context: This is what we show users if they add an update version to their release in an invalid format.

For example, mpm requires format like `R2021aU2` for R2021 update 2. If they put `R2021aUpdate2` or `R2021aSomeNonsense` they will see this.

Reply:
How about this?

Defaulting to the latest update release because ${} is not a valid release name.

Reference page: https://www.mathworks.com/help/matlab/ref/matlabrelease.html